packages/mds-daily/tests/request-handlers.test.ts

```diff
@@ -14,9 +14,9 @@ describe('Request handlers', () => {
         } as any
       }
       await getRawTripData(
-        {
+        ({
           params: { trip_id: 'fake-trip-id' }
-        } as DailyApiRequest,
+        } as unknown) as DailyApiRequest,
```

Comment:
Just curious, why do we need this type annotation now?

Reply:
I regenerated the lock file and TS started complaining about this.
```
Conversion of type '{ params: { trip_id: string; }; }' to type 'Request' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type '{ params: { trip_id: string; }; }' is missing the following properties from type 'Request': get, header, accepts, acceptsCharsets, and 66 more.
```
The recommended fix is to `Add 'unknown' conversion for non-overlapping types`